src/Nethermind/Nethermind.Db.Rocks/Config/DbConfig.cs

```diff
@@ -154,6 +155,7 @@ public class DbConfig : IDbConfig
     public int? StateDbBlockSize { get; set; } = 4 * 1024;
     public bool? StateDbUseDirectReads { get; set; } = false;
     public bool? StateDbUseDirectIoForFlushAndCompactions { get; set; } = false;
+    public bool? StateDbDisableCompression { get; set; } = true;
```

Comment:
This isn't used?

Reply:
No this is used. The `PerTableConfig` class check the field `<dbname>DisableCompression` via reflection. I've checked manually via custom log.